src/rustdoc-json-types/lib.rs

```diff
@@ -37,8 +37,8 @@ pub type FxHashMap<K, V> = HashMap<K, V>; // re-export for use in src/librustdoc
 // will instead cause conflicts. See #94591 for more. (This paragraph and the "Latest feature" line
 // are deliberately not in a doc comment, because they need not be in public docs.)
 //
-// Latest feature: rustdoc JSON: Don't apply #[repr] privacy heuristics
-pub const FORMAT_VERSION: u32 = 46;
+// Latest feature: Pretty printing of inline attributes changed
+pub const FORMAT_VERSION: u32 = 48;
```

Comment:
Uh, we skipped 47. Oh well, not much to be done about it now.

Reply:
@GuillaumeGomez: when you add the (tidy?) check for updating the number and comment in tandem, could you also add a check that the number is being increased by one?

Reply:
I opened https://github.com/rust-lang/rust/pull/142677 for that. But that's a good point. Added to my TODO list.

Reply:
https://github.com/rust-lang/rust/pull/138165#discussion_r2151062261, earlier it was thaught that someone else was going to do 46->47, to this should do 47->48. I think the takeaways are:

a) Don't try to be too clever and do a double bounce
b) #142601 *should* fix this.